doc/whats_new/v0.21.rst

```diff
@@ -134,6 +134,10 @@ Support for Python 3.4 and below has been officially dropped.
   :issue:`12177` by :user:`Sergey Feldman <sergeyf>` :user:`Ben Lawson
   <benlawson>`.
 
+- |Feature| Added the class, :class:`impute.KNNImputer`, to impute missing 
+  values using k-Nearest Neighbors. :issue:`2989` by 
```

Comment:
This should be a PR number not an issue number.

Also don't be afraid to add a link to your own PR ;)

Also ~~the class,~~? (above)